cloudify/workflows/workflow_context.py

```diff
@@ -672,6 +675,9 @@ def _execute_operation(self,
         final_kwargs = self._merge_dicts(merged_from=kwargs,
                                          merged_into=operation_properties,
                                          allow_override=allow_kwargs_override)
+        operation_properties_types = op_struct.get('inputs_types')
+        if operation_properties_types:
```

Comment:
either `is not None` here, or handle None in `_validate_types` (currently it requires a dict, not optional[dict])